src/cartridges/int_adyen_overlay/cartridge/scripts/util/AdyenHelper.ds

```diff
@@ -472,7 +472,7 @@ var __AdyenHelper: Object = {
      * Checks if payment method is blocked
      */
     isMethodTypeBlocked : function(methodType) {
-        var blockedMethods = ["bcmc", "bcmc_mobile_QR","scheme", "cup", "applepay", "wechatpay", "wechatpay_pos", "wechatpaySdk", "wechatpayQr"];
+        var blockedMethods = ["bcmc_mobile_QR", "applepay", "wechatpay", "wechatpay_pos", "wechatpaySdk", "wechatpayQr"];
```

Comment:
I see the bcmc, scheme and cup has been removed - does this mean that you render these as separate payment methods or you handle them somewhere else?

Reply:
Scheme and bcmc are now handled separately. Cup still need to be hidden on the frontend as there is no component yet.